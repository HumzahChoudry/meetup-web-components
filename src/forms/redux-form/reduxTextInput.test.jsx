import React from 'react';
import { shallow } from 'enzyme';
import ReduxTextInput from './ReduxTextInput';

describe('ReduxTextInput', function() {

	const formAttrs = {
		input: {
			label: 'Super Hero',
			name: 'superhero',
			value: 'Wonder Woman and Robin',
			maxLength: 20,
			required: true
		},
		meta: {
			error: 'Did you mean Batman and Robin?'
		}
	};

	it('renders a TextInput component with expected attributes from mock data', () => {
		const component = shallow(<ReduxTextInput {...formAttrs} />);

		expect(component).toMatchSnapshot();

	});

});
