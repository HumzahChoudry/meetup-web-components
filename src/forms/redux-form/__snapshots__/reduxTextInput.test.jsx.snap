// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReduxTextInput renders a TextInput component with expected attributes from mock data 1`] = `
<TextInput
  error="Did you mean Batman and Robin?"
  label="Super Hero"
  maxLength={20}
  name="superhero"
  required={true}
  value="Wonder Woman and Robin"
/>
`;
